cmd/swapd/main.go

```diff
@@ -433,6 +434,7 @@ func (d *daemon) make(c *cli.Context) error { //nolint:gocyclo
 		return err
 	}
 	d.host = host
+	rhost := setupRelayerNetwork(d.ctx, host.P2pHost())
```

Comment:
Not critical: I like terse variable names in short functions, but in a 181 line function should we call this relayHost or relayerHost to avoid confusion with routedHost?